test/e2e/test_catalogue_item.py

```diff
@@ -109,13 +111,55 @@
 }
 
 
+# pylint: disable=duplicate-code
+def _post_units(test_client):
+    """Utility function for posting all mock units defined at the top of this file"""
+
+    response = test_client.get("/v1/units")
+    units = response.json()
+
+    if response.json() == []:
```

Comment:
Why is this needed?

Reply:
this is needed if `_post_catalogue_category_with_units` function is ran twice